gradle/libs.versions.toml

```diff
@@ -39,8 +39,8 @@ pluginPackages-jetbrains-grammarkit = "org.jetbrains.intellij.plugins:gradle-gra
 pluginPackages-jetbrains-intellij = "org.jetbrains.intellij.plugins:gradle-intellij-plugin:1.7.0"
 # @pin because as per https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library:
 #      "If a plugin supports multiple platform versions, it must (...) target the lowest bundled stdlib version"
-#      which is different in our case (as this is the Kotlin version bundled into IntelliJ 2021.3).
-pluginPackages-jetbrains-kotlin = "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10"
+#      which is different in our case (as this is the Kotlin version bundled into IntelliJ 2022.1).
+pluginPackages-jetbrains-kotlin = "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.20"
```

Comment:
Pls remove the `@pin` from `pluginPackages-aspectj-postCompileWeaving`

Reply:
pin must stay; comment updated